hack/update/ingress_version/update_ingress_version.go

```diff
@@ -84,7 +85,9 @@ func main() {
 }
 
 func LatestControllerTag(ctx context.Context) (string, error) {
```

Comment:
if this is a Docker Hub Specific Func, maybe it could be re-used across all automations ? hopefully this code is not duplicated?

Reply:
We have a common version that the rest of the scripts use, however we need a one-off for this one because their tags follow the format `controller-vX.X.X`. Our common func checks if the tag matches semver and ignores ones that don't.